### PR TITLE
seperate people_recognition from image_recognition

### PR DESCRIPTION
**Pull request by MatthijsBurgh**

### CODE REVIEW

File: launch/state_machines/challenge_open.launch [hunk 1]
```diff
@@ -16,14 +18,22 @@
...
+            <arg name="machine" value="sergio2"/>
+            <arg name="robot_name" value="$(arg robot_name)" />
+        </include>
+
         <!-- Joke -->
         <include file="$(find robot_launch_files)/launch/essentials/joke.launch" >
             <arg name="machine" value="sergio2" />
         </include>
 
-        <include file="$(find telegram_robot)/launch/conversation_engine_bot.launch" >
```
Comment (PetervDooren) [inline launch/state_machines/challenge_open.launch:24]: Why remove the telegram interface?

File: launch/state_machines/challenge_help_me_carry.launch [hunk 2]
```diff
@@ -2,27 +2,40 @@
...
+            <arg name="machine" value="sergio2"/>
+            <arg name="config" value="world_modeling/world_model_help_me_carry.yaml"/>
+        </include>
 
-    		<!-- WORLD MODEL -->
-    		<include file="$(find robot_launch_files)/launch/world_modeling/world_model.launch">
-    			<arg name="machine" value="sergio2"/>
-    			<arg name="config" value="world_modeling/world_model_help_me_carry.yaml"/>
-    		</include>
+        <!-- IMAGE RECOGNITION -->
```
Comment (PetervDooren) [inline launch/state_machines/challenge_help_me_carry.launch:25]: Is image recognition needed for help me carry?

File: launch/state_machines/challenge_set_a_table.launch [hunk 3]
```diff
@@ -19,6 +21,17 @@
             <arg name="machine" value="sergio2"/>
         </include>
 
+        <!-- IMAGE RECOGNITION -->
+        <include file="$(find robot_launch_files)/launch/world_modeling/image_recognition.launch">
+            <arg name="machine" value="sergio2"/>
+        </include>
+
+        <!-- PEOPLE RECOGNITION -->
```
Comment (PetervDooren) [inline launch/state_machines/challenge_set_a_table.launch:29]: Is people recognition required for set a table?

File: launch/state_machines/challenge_spr.launch [hunk 4]
```diff
@@ -2,25 +2,38 @@
...
+            <arg name="machine" value="sergio2"/>
+        </include>
+
+        <!-- WORLD MODEL -->
+        <include file="$(find robot_launch_files)/launch/world_modeling/world_model.launch">
+            <arg name="machine" value="sergio2"/>
+            <arg name="config" value="world_modeling/world_model_spr.yaml"/>
+        </include>
+
+        <!-- IMAGE RECOGNITION -->
```
Comment (PetervDooren) [inline launch/state_machines/challenge_spr.launch:25]: Is image recognition needed for challenge spr?

File: launch/state_machines/challenge_storing_groceries.launch [hunk 5]
```diff
@@ -19,6 +21,17 @@
             <arg name="machine" value="sergio2"/>
         </include>
 
+        <!-- IMAGE RECOGNITION -->
+        <include file="$(find robot_launch_files)/launch/world_modeling/image_recognition.launch">
+            <arg name="machine" value="sergio2"/>
+        </include>
+
+        <!-- PEOPLE RECOGNITION -->
```
Comment (PetervDooren) [inline launch/state_machines/challenge_storing_groceries.launch:29]: People recognition is not needed for storing groceries.

File: launch/state_machines/gmapping.launch [hunk 6]
```diff
@@ -21,6 +23,17 @@
             <arg name="config" value="world_modeling/world_model_gmapping.yaml"/>
         </include>
 
+        <!-- IMAGE RECOGNITION -->
```
Comment (PetervDooren) [inline launch/state_machines/gmapping.launch:26]: Image and people recognition is not needed in gmapping

**Review by PetervDooren — DISMISSED** [review]

**Comment by MatthijsBurgh** [comment]
Decided to run everything all the time, to have a fully functional robot object.

File: launch/state_machines/challenge_open.launch (hunk 1 above)
Comment (MatthijsBurgh) [reply launch/state_machines/challenge_open.launch:24]: Outdated, shouldn't be used anymore.

**Comment by MatthijsBurgh** [comment]
ping ;)

**Review by LoyVanBeek — APPROVED** [review]

**Pull request merged**